packages/cli/__tests__/daemon/__unit__/DaemonDecider.unit.test.ts

```diff
@@ -240,12 +240,16 @@ describe("DaemonDecider tests", () => {
 
         let expectedCommChannel: string;
         if (process.platform === "win32") {
-            expectedCommChannel = `\\\\.\\pipe\\${os.userInfo().username}\\ZoweDaemon`;
+            expectedCommChannel = `\\\\.\\pipe\\${os.userInfo().username.toLocaleLowerCase()}\\ZoweDaemon`;
         } else {
             expectedCommChannel = path.join(os.homedir(), ".zowe/daemon/daemon.sock");
         }
 
-        expect((daemonDecider as any).mSocket).toEqual(expectedCommChannel);
+        if (process.platform === "win32") {
```

Comment:
I do not see any difference between the "if" and "else" actions?

Reply:
Ah thanks for spotting this Gene. Initially I had the `win32` branch checking for the lowercase channel path, but after updating the logic to use a common variable, we no longer need the if/else. I've addressed this in my upcoming commit 👍